### PR TITLE
Task 4 1

### PR DESCRIPTION
**Pull request by DenKov07**

### CODE REVIEW

File: chapter04/Task_4_1.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# The program displays a string of numbers with the specified start, end , and step
```
Comment (satanevsky) [inline chapter04/Task_4_1.py]: Unnecessary space in "end ,"

Comment (satanevsky) [inline chapter04/Task_4_1.py:1]: Unnecessary space in "end ,"

**Review by satanevsky — APPROVED** [review]
You have 2 files that aren't needed there, please delete

File: chapter04/Task_4_1.py (hunk 1 above)
Comment (DenKov07) [reply chapter04/Task_4_1.py:1]: ?

**Pull request closed without merging**